pkgs/by-name/an/annotator/package.nix

```diff
@@ -1,27 +1,29 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, pkg-config
-, meson
-, ninja
-, vala
-, wrapGAppsHook3
-, desktop-file-utils
-, libgee
-, pantheon
-, libxml2
-, libhandy
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pkg-config,
+  meson,
+  ninja,
+  vala,
+  wrapGAppsHook3,
+  desktop-file-utils,
+  libgee,
+  pantheon,
+  libxml2,
+  libhandy,
+  libportal-gtk4,
 }:
 
 stdenv.mkDerivation rec {
   pname = "annotator";
-  version = "1.2.1";
+  version = "2.0.0";
 
   src = fetchFromGitHub {
     owner = "phase1geo";
     repo = "annotator";
     rev = version;
-    hash = "sha256-VHvznkGvrE8o9qq+ijrIStSavq46dS8BqclWEWZ8mG8=";
+    hash = "sha256-mv3fMlYB4XcAWI6O6wN8ujNRDLZlX3ef/gKdOMYEHq0=";
   };
 
   nativeBuildInputs = [
```

Comment:
`wrapGAppsHook4` instead of 3, since we are using gtk4 now

Reply:
it still wants `gtk+-3.0` as a runtime dependency, I don't know how to fix that

```
error: builder for '/nix/store/3clp5sp8fdx2903fcbyxraj0gwhf4rqz-annotator-2.0.0.drv' failed with exit code 1;
       last 25 log lines:
       > Source dir: /build/source
       > Build dir: /build/source/build
       > Build type: native build
       > Project name: com.github.phase1geo.annotator
       > Project version: 2.0.0
       > C compiler for the host machine: gcc (gcc 13.3.0 "gcc (GCC) 13.3.0")
       > C linker for the host machine: gcc ld.bfd 2.43.1
       > Vala compiler for the host machine: valac (valac 0.56.17)
       > Host machine cpu family: x86_64
       > Host machine cpu: x86_64
       > Found pkg-config: YES (/nix/store/f1gamyykr0f8n4afy910ass2jx2g9zkv-pkg-config-wrapper-0.29.2/bin/pkg-config) 0.29.2
       > Build-time dependency gio-2.0 found: YES 2.82.1
       > Program /nix/store/pq7p4bqr2wimdp1qp01q10gbvasd55hk-glib-2.82.1-dev/bin/glib-compile-resources found: YES (/nix/store/pq7p4bqr2wimdp1qp01q10gbvasd55hk-glib-2.82.1-dev/bin/glib-compile-resources)
       > Configuring Config.vala using configuration
       > Program msgfmt found: YES (/nix/store/6xqkmada5jpx49dbrl9zjgzslcxzzr17-gettext-0.21.1/bin/msgfmt)
       > Program msginit found: YES (/nix/store/6xqkmada5jpx49dbrl9zjgzslcxzzr17-gettext-0.21.1/bin/msginit)
       > Program msgmerge found: YES (/nix/store/6xqkmada5jpx49dbrl9zjgzslcxzzr17-gettext-0.21.1/bin/msgmerge)
       > Program xgettext found: YES (/nix/store/6xqkmada5jpx49dbrl9zjgzslcxzzr17-gettext-0.21.1/bin/xgettext)
       > Did not find CMake 'cmake'
       > Found CMake: NO
       > Run-time dependency gtk+-3.0 found: NO (tried pkgconfig)
       >
       > meson.build:41:10: ERROR: Dependency "gtk+-3.0" not found, tried pkgconfig
       >
       > A full log can be found at /build/source/build/meson-logs/meson-log.txt
       For full logs, run 'nix log /nix/store/3clp5sp8fdx2903fcbyxraj0gwhf4rqz-annotator-2.0.0.drv'.
```

Reply:
Add `gtk3` to `buildInputs`